foxy/ci-nightly-extra-rmw-release.yaml

```diff
@@ -28,6 +28,7 @@ jenkins_job_label: ci-agent
 jenkins_job_priority: 47
 jenkins_job_schedule: 15 23 * * *
 jenkins_job_timeout: 300
+package_selection_args: '--packages-ignore test_communication'
```

Comment:
Now that we skip multi-rmw tests, I think we don't need to ignore `test_communication`

Reply:
Good catch - absolutely: 08532c1192515e98107ae78ca0eb353b7866a263.